repository/src/test/java/org/alfresco/util/schemacomp/SchemaReferenceFileTest.java

```diff
@@ -25,47 +25,47 @@
  */
 package org.alfresco.util.schemacomp;
 
-
-import static org.junit.Assert.fail;
+import static org.junit.Assert.assertTrue;
 
 import java.io.ByteArrayOutputStream;
 import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
 
 import org.alfresco.repo.domain.schema.SchemaBootstrap;
 import org.alfresco.test_category.OwnJVMTestsCategory;
 import org.alfresco.util.ApplicationContextHelper;
 import org.alfresco.util.testing.category.DBTests;
-import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.springframework.context.support.ClassPathXmlApplicationContext;
 
 /**
- * Test intended for use in the continuous integration system that checks whether the
- * schema reference file (for whichever database the tests are being run against)
- * is in sync with the actual schema. If the test fails (and the schema comparator is
- * in working order) then the most likely cause is that a new up-to-date schema reference file
- * needs to be created.
+ * Test intended for use in the continuous integration system that checks
+ * whether the schema reference file (for whichever database the tests are being
+ * run against) is in sync with the actual schema. If the test fails (and the
+ * schema comparator is in working order) then the most likely cause is that a
+ * new up-to-date schema reference file needs to be created.
  * <p>
  * Schema reference files are created using the {@link DbToXML} tool.
  * <p>
- * Note: if no reference file exists then the test will pass, this is to allow piece meal
- * introduction of schmea reference files.
+ * Note: if no reference file exists then the test will pass, this is to allow
+ * piece meal introduction of schmea reference files.
  * 
  * @see DbToXML
  * @author Matt Ward
  */
-@Category({OwnJVMTestsCategory.class, DBTests.class})
+@Category({ OwnJVMTestsCategory.class, DBTests.class })

```

Comment:
Doubt this is a big deal but I'd remove the extra spaces here (haven't seen this occur anywhere else). I think this was cause by a missed automatic reformatting of the overall code.

Reply:
Yes it's probably the formatter, with an "organize import" as well.